pints/_mcmc/__init__.py

```diff
@@ -878,6 +932,37 @@ def samplers(self):
         """
         return self._samplers
 
+    def _sample_x0(self, active, evaluator):
+        """
+        Samples an initial starting point for the chains using the supplied
+        `init` function.
+        """
+        initialised_finite = False
+        init_fn = self._init_fn
+        current_active = list(active)
```

Comment:
Are they not all active by default?

Reply:
Nope. Some samplers have intermediate steps, so we return a list of those 'active' samplers still waiting for their intermediate iterations to complete.

Reply:
On the first step though :D 
But I'm happy for this to stay in